base-theme/assets/fonts/material-design-icons/README.md

```diff
@@ -0,0 +1,60 @@
+# Material Icons' Subsetting
+
+This directory holds Material Icons used within OCW websites.
+
+We subset material icons' fonts to make them smaller for delivery. By "subsetting a font" we simply mean removing unnecessary characters from the font face.
+
+Each time the set of icons changes, new subset font assets must be created. To do so, read the following sections.
+
+## Prerequisites
+
+* You must have the dependencies from the main project installed.
+
+* Install [fonttools](https://github.com/fonttools/fonttools) utility.
```

Comment:
It is recommended to use subset-font (based on Wasm) so that you don't need to install fonttools.

https://www.npmjs.com/package/subset-font

Reply:
I tried this at first but the resulting subset didn't work. Maybe it was because I wasn't including a default set of glyphs which I am doing now.

Thanks for the review. I'll try it out with the default glyphs to see if that was the problem.

Reply:
For testing purposes, you can try installing `hb-subset`, which is the core behind subset-font.

```
brew install harfbuzz
hb-subset --help-all
```

Reply:
Thank you for the references. I managed to make a subset using `subset-font` and `hb-subset`.

I think we might not be able to use `subset-font`, as it does not provide a way to disable layout closure.

`subset-font` produces a result equivalent to 
```
hb-subset MaterialIcons-Regular.ttf <icons> -o out.ttf
```

But what I want is
```
hb-subset MaterialIcons-Regular.ttf <icons> -o out.ttf --no-layout-closure
```

There's a significant difference in the output sizes of these commands.

If I understand correctly, the rationale behind this is as follows: To include ligatures in the subset, we must include the individual glyph components that make up the ligature. In our case, these glyph components are `_a-z0-9`. If layout closure is enabled, all the glyphs that can be formed by these glyph components are also included in the subset. This causes unnecessary glyphs to be included.

I think I'll create a feature request in `subset-icon`. I'd definitely prefer using it when the layout closure option is added.

Reply:
https://github.com/papandreou/subset-font/issues/22